python/cudf/cudf/_lib/datetime.pyx

```diff
@@ -86,6 +86,33 @@ def ceil_datetime(Column col, object field):
     return result
 
 
+def floor_datetime(Column col, object field):
+    cdef unique_ptr[column] c_result
+    cdef column_view col_view = col.view()
+
+    with nogil:
+        # https://pandas.pydata.org/pandas-docs/version/0.25.0/reference/api/pandas.Timedelta.resolution.html
```

Comment:
This URL might be better since it points to a newer pandas version, and the name has changed from `resolution` to `resolution_string`.
```suggestion
        # https://pandas.pydata.org/docs/reference/api/pandas.Timedelta.resolution_string.html
```

Reply:
Thanks Bradley for catching this!